README.md

```diff
@@ -1,2 +1,12 @@
 # dns_recon
 DNS recon based on domain name.
+
+## Getting Started
+
+- Create virtual environment
+  - `python -m venv env`
+  - `.\Scripts\Activate.ps1`
```

Comment:
Can we distinguish that this is how to activate the virtual environment for Windows and add a line for Linux?

The line for linux would be:
`source` env/bin/activate`

Reply:
Yes, over a year later...we can do this, lol